inc/namespace.php

```diff
@@ -29,7 +29,10 @@ function setup() {
  * @return array
  */
 function get_client_side_data() : array {
-
+	// Return empty object if preview.
+	if ( is_preview() ) {
```

Comment:
We may still need the data for other things like the AB test components. I'll add my suggestion for the approach in the main review comment.